### DISCUSSION TITLE
Client Reference Implementation - Area for Feedback

### DISCUSSION THREAD

**Discussion by evalstate** [modelcontextprotocol/.github#52] [General]
### Pre-submission Checklist

- [X] I have verified that this discussion would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions to avoid duplicates

### Discussion Topic

MCP was released with Claude Desktop as the reference implementation (for example, it is used to lead the discussion in quick reference).

However - I think there needs to be a separate place for discussing/raising issues with Claude Desktop [CD] specifically.

CD is obviously important as Anthropic's commercial offering and lead MCP platform - but I think there needs to be  very clear distinction between MCP discussions, ideas for Client implementations and the CD application itself. 

At the very least, I think a separate discussion area here, with guidance on the role of Claude Desktop as a reference platform would be beneficial.

**Comment by neptune-v** [comment]
https://github.com/orgs/modelcontextprotocol/discussions/36#discussioncomment-11389347

**Reply by ArsenicBismuth** [reply]
If people want temporary example, the thread inside has my example for Brave server.

**Comment by jspahrsummers** [comment]
Agreed that it's a bit messy right now! For the time being, we're willing to help people set up MCP + Claude Desktop through discussions and issues in @modelcontextprotocol—particularly while there are a handful of bugs where it's not clear which code is at fault—but we'll be aiming to transition this to a separate support channel later.
